client/my-sites/plans-features-main/index.jsx

```diff
@@ -379,6 +388,44 @@ export class PlansFeaturesMain extends Component {
 		return false;
 	}
 
+	renderProductsSelector() {
+		if ( ! isEnabled( 'plans/jetpack-backup' ) ) {
+			return null;
+		}
+
+		const { intervalType, siteId, translate } = this.props;
+		const products = [
```

Comment:
Feels weird to define this on every re-render. Let's make sure to define it somewhere where it will be defined only once.